zerver/tests/test_import_export.py

```diff
@@ -219,7 +233,7 @@ def read_file(fn: str) -> Any:
         result = {}
         result['realm'] = read_file('realm.json')
         result['attachment'] = read_file('attachment.json')
-        result['message'] = read_file('message.json')
+        result['message'] = read_file('messages-000001.json')
```

Comment:
Was there a reason for this name change?

Reply:
Yes, all messages are produced batch wise in the export with the naming `messages-000001.json`, `messages-000002.json` and onwards ([reference](https://github.com/zulip/zulip/blob/master/zerver/lib/export.py#L921)). The import script can read the files only with this naming system, hence for the import tests I have changed the name.

Reply:
Separated this in a new commit.